src/util/customTransforrmationsStore-v1.js

```diff
@@ -29,14 +29,14 @@ async function getTransformationCodeV1(versionId) {
     const response = await fetchWithProxy(url);
 
     responseStatusHandler(response.status, 'Transformation', versionId, url);
-    stats.increment('get_transformation_code.success', tags);
```

Comment:
'.' is not allowed as part of metric name in prometheus. Using success as a tag. This condenses two metrics (get_transformation_code.success, get_transformation_code.error) into one.